libopenage/datastructure/tests.cpp

```diff
@@ -192,4 +191,146 @@ void constexpr_map() {
 	TESTEQUALS(cmap.get(42), 9001);
 }
 
-}}} // openage::datastructure::tests
+/**
+ * A simple class that can be move-constructed but not copy-constructed
+ */
+class MoveOnly{
+	private:
+	int m_data;
```

Comment:
we place the access labels at the same indent level as the `class` statement.
our member names are (at least until now) never named `m_`, since we always access them with `this->`.
for consistency reasons it would be nice if you removed the prefix (and access the data in `get()` with `this->data` then).

Reply:
OK what about static variables then? Should I also remove the `s_` ?

Reply:
I'd say yes, but we don't have a strict rule there yet.